schema.sql

```diff
@@ -0,0 +1,52 @@
+create schema if not exists appdata;
+
+CREATE TABLE IF NOT EXISTS appdata.user
+(
+  user_id bigserial NOT NULL 
+  , email varchar(50) UNIQUE NOT NULL
+  , password varchar(10) NOT NULL
+  , name varchar(30) NOT NULL
+  , age smallint 
+  , gender varchar(3)
+  , city varchar(20)
+  , country varchar(20)
+  , browser varchar(10)
+  , is_developer boolean
+  , created_on timestamp without time zone NOT NULL
+  , last_login timestamp without time zone NOT NULL
+  , PRIMARY KEY (user_id)
```

Comment:
I would suggest that we have a primary key of (uuid, email, username) instead. user_id -> uuid, name -> username

Reply:
done currently primary key is composite (uuid,username)